docs/tutorials/CustomModels.ipynb

```diff
@@ -193,7 +193,7 @@
     "\n",
     "- What was \"sample_image\" in the radial_model function? This is an object for the image that we are currently sampling. You shouldn't need to do anything with it except get the pixelscale.\n",
     "- what else is in \"ap.utils\"? Lots of stuff used in the background by AstroPhot. For now the organization of these is not very good and sometimes changes, so you may wish to just make your own functions for the time being.\n",
-    "- Why the weird way to access the parameters? The self\\[\"variable\"\\].value format was settled on for simplicity and generality. it's not perfect, but it works.\n",
+    "- Why the weird way to access the parameters? The self\\[\"variable\"\\].value format was settled on for simplicity and generality.  It's not perfect, but it works.\n",
```

Comment:
Just noticed these `.ipynb` notebooks have been removed from the AstroPhot repo to go to their own repo. You can just delete the notebooks